scripts/check-migration-numbering.sh

```diff
@@ -34,7 +53,11 @@ for i in "${!prefixes[@]}"; do
   base_filename=$(ls "$migrations_path/${prefixes[$i]}"_*.up.sql | \
       sed -E 's/\.up\.sql//')
 
-  if [ ! -f "$base_filename.down.sql" ]; then
+  # Error if the .up.sql is an SQL migration, but we're missing the
+  # corresponding .down.sql. This doesn't apply if the .up.sql file is a code
+  # migration.
+  if is_sql_migration "$base_filename.up.sql" && \
+      [ ! -f "$base_filename.down.sql" ]; then
```

Comment:
Updated:

Do we need to include this commit? Is it required for your end goal?

We don’t often have no-op down migrations, but IMO it’s fine to include them. It keeps the migration sequence contiguous, and IMO it’s cleaner to always provide a down file for every up, even if the down is empty.